bugbug/repository.py

```diff
@@ -91,14 +92,15 @@ def hg_log(repo_dir):
     revs = []
     for rev in hglib.util.grouper(template.count('\\0'), out):
         posixtime = float(rev[3].split(b'.', 1)[0])
+        
```

Comment:
You added a newline here instead of readding the one you removed :smile:

Reply:
Just a minute. sorry